### ISSUE TITLE
RFC7: Lesson organisations

### ISSUE THREAD

**Issue by k8hertweck** [carpentries/2017Merger#7]
View RFC7 [here](https://github.com/carpentries/2017Merger/blob/master/RFCs/RFC7.md)

**Q1: Do you have any comments or suggestions on the organisation and role of the Lesson Organisations under the umbrella of The Carpentries organisation?** Specifically:

Q1a: responsibilities to The Carpentries and to the community?

Q1b: governance model?

**Comment by JasonJWilliamsNY** [comment]
Q1b: It does not have to be detailed here, but I think its a requirement that the lesson orgs (which I question if they are orgs in #1 ) have as identical as possible governance structures. For me what comes to mind is a board of maintainers with a Chair. I might even think that Chair is on the Carpentries board. Somehow, I would guess the lesson orgs are mostly maintainer driven and may have churn as folks might only give so much for so long. Lesson communities could elect the Chair, and the Carpentries Board could inherit the result of that election - keeping things as community-driven as possible. 

I like baroque things, so maybe there are good examples from other software communities. But I do stand by having very clear ground rules for all of them to keep the Carpentries less of a nightmare.

**Comment by wrightaprilm** [comment]
I'm sort of confused about what a lesson organization is - is that like "Data Carpentry - Ecology" and all the sublessons?

**Comment by k8hertweck** [comment]
@wrightaprilm Software Carpentry and Data Carpentry are intended to be the first Lesson Organizations, although their scope of responsibilities will be substantially reduced from current operations. However, I'm not sure what the specific plans are for Data Carpentry's structure. Maybe @tracykteal can shed some light?

**Comment by tracykteal** [comment]
Data Carpentry will continue to have lessons in multiple domains. So for instance 'ecology' and 'genomics' will continue to be Data Carpentry. Both sets of lessons are focused on the core data skills and perspectives needed to effectively and reproducibly work with data in that domain. 

Where we're thinking something will be a new 'Carpentry' is where it has a different type of mission for its lessons and workshops. For instance Library Carpentry's goal is to help create 'data-savy librarians' https://librarycarpentry.github.io  

As @JasonJWilliamsNY suggests, there will be guidelines for what a Lesson Carpentry can be under the Carpentries umbrella, and those are guidelines that will start to come together with the new joint Steering Committee and comments from the community. Clear ground rules will be very important.

**Comment by wrightaprilm** [comment]
OK, so the idea is that a lesson organization will be Data/Software/Lib Carpentry. For DC, then, within each domain, each lesson has a maintainer or two, and there is a representative for the domain who is tracking the progress and needs, generally, of all the domain lessons. And then one chair is across all domains, and this person is in charge of representing the interests of the whole  LO in discussions across LOs about lesson development and maintenance?

Do I have that right?

**Comment by pitviper6** [comment]
@wrightaprilm That's how I was envisioning it. For instance, from the potential-merger LC standpoint, I could see having the various lessons, each with lesson maintainers. The lesson maintainers have regular meetings to talk about lesson issues, consistent formatting, etc. etc. They choose a chair to be the liaison/report person to the larger Carpentries org.

**Comment by wrightaprilm** [comment]
Right, I like that structure. It would be nice to have someone, like a Ecology lesson council, to bounce ideas off of when I get a pull request that is trickier to resolve. Right now, we don't really have that, and refereeing pull requests ends up getting taken way too far up the chain as a result. This seems both more stable and responsive.

**Comment by pitviper6** [comment]
It's also a good way to avoid duplicating discussion - if a particular issue is discussed and resolved in a certain way for one lesson, they can report that to the other maintainers. If it emerges in another lesson, they know it's already been discussed and can see if that applies to their lesson.

**Comment by ErinBecker** [comment]
Agreed with @pitviper6 and @wrightaprilm about having an advisory group that covers all curricula within a domain. DC is currently testing this model with the genomics curriculum with the goal of expanding it to the Ecology and Geospatial curricula in the near future. @tracykteal

**Comment by kcranston** [comment]
One of the open questions is how much autonomy each Lesson Organization has? Should the Umbrella Org dictate the governance model of the Lesson Orgs, or let each one come up with a structure that works for them? Software / Data / Library are at very different stages - what works for an established group might be too much for another group at the incubator stage.

**Comment by pitviper6** [comment]
@kcranston Interesting question! My hot take is that upper level/reporting structure would have to come into some kind of line with the umbrella org, but as you get to the maintainer level, there can be a more organic structure depending on how the people in that lesson like to work, the important thing being the communications back and forth with the other lesson maintainers and the Carpentries staff/BoD.  I think if the communication is strong, eventually the lesson orgs will figure out the best way to do things from talking to each other, and processes will become more uniform across the lessons.

**Comment by anelda** [comment]
Hi all, I have a question about using lessons across lesson organisations in workshops. For example, we have often used lessons from LC and DC in one workshop. Or we've combined the SWC shell lesson with the DC ecology workshop material. Sometimes we switch out some of the "core" material in exchange (like not doing OpenRefine or SQL when we include shell or other lessons in the DC Ecology workshop). We try to cater to our audience's need. The only place where I've seen a problem in terms of doing this, is where we want to issue certificates - which certificate do people get - a Carpentries one, or a joint DC/SWC or DC/LC or ...? Not sure if this is the place to ask the question, please feel free to move this to the right place. Thanks.

**Comment by wrightaprilm** [comment]
You mean in terms of issuing certificates to learners?  Do we do that? It's sort of unclear to me which lessons in the (for example) Ecology lesson are actually "core". I thought it was just 3 lessons in a domain to qualify. We can teach data cleaning, management, analysis and visualization without teaching OpenRefine or SQL for example. I've used the second afternoon at workshops to do an abbreviated shell + git, which while SQL is really useful for some tasks, I'd argue was a much more immediate benefit to the data management practices of the learners with whom I was actually working than SQL would be. So that workshop had 3 core lessons (spreadsheets, OR, and R) and one non-core.

So that's one question: What is the core for learners? I think another question is "What is the core for instructors, and is it OK for some instructors to not have all the pieces?" Last week on the maintainer call, it was brought up that some instructors aren't comfortable enough with Github to file an issue or a pull request. To me, segments of the contributing membership being unable to contribute in the normal workflow is a big problem. It's my understanding, certification is presently pan-Carpentry, which means that there are instructors being badged to teach SWC workshops that they can't actually teach. I guess this circles back to @kcranston's  question, but to what extent is there autonomy in deciding what core competencies instructors need?

**Comment by ChristinaLK** [comment]
@anelda , we've never mixed material like that, but I've thought about it and wondered what the best approach would be.  My feeling is that it depends what you were trying to teach at the workshop.  Were you teaching people how to analyze data and best practices for that?  Or useful tools and practices for actually writing code?  (Or both?)  The first is the DC mission, the second is the SWC mission.  

This is something I would like to clarify though, as the SWC lesson org taskforce gets kicked off this month.

**Comment by ChristinaLK** [comment]
@wrightaprilm The question of "core" is an important one.  I definitely think that's something that the SWC/DC organizations have and should continue to discuss, so that there are clear guidelines that also allow for some flexibility.  
In terms of instructors, I think we've never actually asked for particular core competencies, we've assumed that if you want to teach it, you'll put in the work to learn about it.  That's maybe worth clarifying.

**Comment by drjwbaker** [comment]
> Onboarding for new lesson organizations: The Carpentries Board will be responsible for assessing alignment of the prospective lesson organization’s mission and their ability to meet the responsibilities described.

I think this needs to be more tightly specified. I presume - perhaps falsely - that The Carpentries are not looking to encourage proliferation from introducing the Lesson Orgs model, rather it is looking to a) sort out how to merge SWC and DC without losing unique branding/sell and b) create a potential space for LC. I guess my point is, what is the difference between having subject/area/domain specific lessons (like DC has) and a lesson org.

**Comment by drjwbaker** [comment]
(otherwise, super happy this is happening!)

**Comment by ChristinaLK** [comment]
@drjwbaker I think proliferation is in fact part of the idea -- that if a community demonstrates enough interest + develops lessons for itself, it could then join the carpentries and get the advantages of workshop organization + instructor training.  
Im working on a task force right now to clarify at least what the lesson org will be for SWC and I see it not so much being lesson materials, but mission.  Teaching best practices for scientific computing vs best practices for data management vs bringing digital skills to librarians are all significantly different missions, regardless of the accompanying lesson stacks.  
 Although now Im thinking that lesson org may not be the best name for that difference.  I will keep thinking on that!

**Comment by npch** [comment]
I think one thing that this thread had made clear to me is that "lesson organization" is definitely the wrong term :-)

**Comment by npch** [comment]
In many ways, this feels like the Apache incubator / project model. Where there is enough community enthusiasm to start a new Carpentry, the Carpentries provide support to develop it. Eventually it matures, and becomes a *(insert new name)* here and receives general support.

I'd like to resurface @JasonJWilliamsNY question of how each "Lesson Organisation" / "Carpentry Collective" formally interacts from a governance / operations / reporting / oversight view with The Carpentries.

One model I have seen in other organisations is that the "chair" (or other nominated person) is an ex-officio member of the Board of Directors (effectively they are a member of staff of "The Carpentries" by virtue of their position leading a "lesson organisation").

**Comment by npch** [comment]
What autonomy do Lesson Organisations have over budgets? Is it possible for them to request a budget from The Carpentries and manage it themselves?

**Comment by drjwbaker** [comment]
> @drjwbaker I think proliferation is in fact part of the idea

@ChristinaLK I should clarify: I meant I assumed Carpentries would be keen to avoid rapid proliferation in terms of lots of new small Carpentries or - in addition - fragmentation of the existing SWC/DC into smaller communities. Either way, I feel the guidelines on this should reflect what Carpentries envisages it can support in - say - year 1, year 2, year 5, with room for revision as time develops. As it stands it - for me at least - is slightly under-specified and therefore a little vanilla.

**Comment by tracykteal** [comment]
@npch there hasn't been much discussion on budgets yet, but an important point. It will be important that Lesson Organizations can bring in funding for work on those specific projects, so have some budget autonomy.

**Comment by ChristinaLK** [comment]
Ah, yes @drjwbaker, you're right there, and I agree.

**Comment by gvwilson** [comment]
I think that every Lesson Organization (or whatever they're finally called) should be required to have the same internal governance structure - anything else will probably make decision-making unmanageably difficult.
